securedrop/journalist.py

```diff
@@ -114,7 +114,10 @@ def login():
             login_flashed_msg = "Login failed."
 
             if isinstance(e, LoginThrottledException):
-                login_flashed_msg += " Please wait at least 60 seconds before logging in again."
+                login_flashed_msg += (
+                    " Please wait at least {} seconds "
```

Comment:
You can get rid of this leading space. Or not. Anyway, good practice of not hard-coding here :+1: